src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.scss

```diff
@@ -1,4 +1,4 @@
-#dropdown-label {
+.dropdown-label {
```

Comment:
I think its preferable to use the gov-uk styles `govuk-!-font-weight-bold`

Reply:
https://design-system.service.gov.uk/styles/typography/#font fyi

Reply:
Thank you @kuda1992 I have now made the changes.